build.gradle

```diff
@@ -46,6 +52,10 @@ dependencies {
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
 }
 
+shadowJar {
+    mergeServiceFiles()
+}
```

Comment:
Yanked from this thread
https://stackoverflow.com/questions/55484043/how-to-fix-could-not-find-policy-pick-first-with-google-tts-java-client